Dockerfile

```diff
@@ -41,7 +41,9 @@ EXPOSE  9292
 
 ## now lists
 #CMD ls -l /usr/local/config/default/*; ls -l /usr/local/config/cred/*; rackup --host 0.0.0.0 -p 9292
-CMD cp /usr/local/config/default/default.yml /app; cp /usr/local/config/cred/ggb-cpm-dev-3bc39c2ec7f7.json /Users/dlhaines/dev/BITBUCKET/GoogleAPIDemo/discussions-dev/GGB-CPM-Dev-3bc39c2ec7f7.json; rackup --host 0.0.0.0 -p 9292
+#CMD thin start -R config.ru -p 3000
+#CMD cp /usr/local/config/default/default.yml /app; cp /usr/local/config/cred/ggb-cpm-dev-3bc39c2ec7f7.json /Users/dlhaines/dev/BITBUCKET/GoogleAPIDemo/discussions-dev/GGB-CPM-Dev-3bc39c2ec7f7.json; rackup --host 0.0.0.0 -p 9292
+CMD cp /usr/local/config/default/default.yml /app; cp /usr/local/config/cred/ggb-cpm-dev-3bc39c2ec7f7.json /Users/dlhaines/dev/BITBUCKET/GoogleAPIDemo/discussions-dev/GGB-CPM-Dev-3bc39c2ec7f7.json; thin start -R config.ru --address 0.0.0.0 --port 9292
```

Comment:
It looks like there are several unnecessary comments left in the code. There is no test plan on the JIRA. Also, it looks like your CMD line is connected via ';' I think what this means is, that if any of the commands fail, the Docker process will just continue on to the next one. If you want to link them to make them all one CMD line you can chain them with '&&' I believe (confirm with Chris).  Furthermore, I think your CMD is just supposed to be something to kick off your process. In my opinion it should just be 'thin start -R config.ru --address 0.0.0.0 --port 9292' and the copies being done prior should be done somewhere else (again, refer to Chris).